src/biogeophys/TemperatureType.F90

```diff
@@ -1215,6 +1215,17 @@ subroutine InitAccBuffer (this, bounds)
        call init_accum_field (name='GDD10', units='K', &
             desc='growing degree-days base 10C from planting', accum_type='runaccum', accum_period=not_used,  &
             subgrid_type='pft', numlev=1, init_value=0._r8)
+       
+       ! 20-year running means (20*365 days)
+       call init_accum_field (name='GDD020', units='K', &
+            desc='20-year running mean of growing degree days base 0C from planting', accum_type='runmean', accum_period=-20*365, &
+            subgrid_type='pft', numlev=1, init_value=0._r8)
+       call init_accum_field (name='GDD820', units='K', &
+            desc='20-year running mean of growing degree days base 8C from planting', accum_type='runmean', accum_period=-20*365, &
+            subgrid_type='pft', numlev=1, init_value=0._r8)
+       call init_accum_field (name='GDD1020', units='K', &
+            desc='20-year running mean of growing degree days base 10C from planting', accum_type='runmean', accum_period=-20*365, &
+            subgrid_type='pft', numlev=1, init_value=0._r8)
```

Comment:
Should this initialization be skipped when restarting?

Reply:
No. An exact restart test, `ERS_Ly5_P144x1.f10_f10_mg37.IHistClm51BgcCrop.cheyenne_intel.clm-cropMonthOutput`, passed `COMPARE_base_rest` as of 574f3b271.

Reply:
- [ ]  Ensure that `GDD*20` fields are actually being written in that test.